utils/notebook_utils.py

```diff
@@ -48,6 +48,19 @@ def device_widget(default="AUTO", exclude=None, added=None, description="Device:
     return device
 
 
+def model_widget(default="Phi-4", description="Model:"):
```

Comment:
please move this widget to notebook. notebook utils is common utilities shared across all notebooks, while your code is related to specific models and specific notebook

Reply:
Done